android/sdk/src/main/java/com/taobao/weex/ui/component/WXComponent.java

```diff
@@ -1682,12 +1684,29 @@ public void setBackgroundImage(@NonNull String bgImage) {
       getOrCreateBorder().setImage(shader);
     }
   }
+  public boolean shouldCancelHardwareAccelerate() {
+    IWXConfigAdapter adapter = WXSDKManager.getInstance().getWxConfigAdapter();
+    boolean cancel_hardware_accelerate = false;
+    if (adapter != null) {
+      cancel_hardware_accelerate = Boolean.parseBoolean(adapter
```

Comment:
Add try catch, there is no way to guarantee that the server would return `true` or `false`, maybe it returns something can't convert to `boolean`